src/tiled/filteredit.cpp

```diff
@@ -33,6 +33,17 @@ FilterEdit::FilterEdit(QWidget *parent)
 
 bool FilterEdit::event(QEvent *event)
 {
+    if (event->type() == QEvent::KeyPress) {
```

Comment:
Why is this change here, @bjorn? I don't think this PR uses FilterEdit anywhere...